<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_overdue_book_ui</class>
 <widget class="QDialog" name="view_overdue_book_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overdue Books</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CBE6C7;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#122C4F;
color:#CBE6C7;</string>
   </property>
   <property name="text">
    <string>OVERDUE BOOKS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="overdue_book_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>460</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gloucester MT Extra Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #CBE6C7;
    color: #122C4F;
    gridline-color: #122C4F;
}

QHeaderView::section {
    background-color: #CBE6C7;
    color: #122C4F;
    padding: 4px;
}

QTableCornerButton::section {
    background-color: #CBE6C7;
}

QAbstractScrollArea::corner {
    background: #CBE6C7;
}
</string>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
